gh_release/ECHAM5_model.xml

```diff
@@ -0,0 +1,271 @@
+<?xml version="1.0"?>
+<metadata>
+  <idinfo/>
+  <citation>
+    <citeinfo>
+      <origin>Alison P Appling</origin>
```

Comment:
yeah, I haven't gone through the real effort yet of making the changes. will be doing that today.